app/models/repository.rb

```diff
@@ -23,14 +23,16 @@ class Repository < ApplicationRecord
   class << self
 
     def remove_suse_repos_without_tokens!
-      where(auth_token: nil).where('external_url LIKE ?', 'https://updates.suse.com%').delete_all
+      where(auth_token: nil)
+        .where("external_url LIKE 'https://updates.suse.com%' OR external_url LIKE 'https://dl.suse.com%'")
+        .delete_all
     end
 
     # Mangles remote repo URL to make a nicer local path, see specs for examples
     def make_local_path(url)
       uri = URI(url)
       path = uri.path.to_s
-      path.gsub!(%r{^/repo}, '') if (uri.hostname == 'updates.suse.com')
+      path.gsub!(%r{^/repo}, '') if (uri.hostname == 'updates.suse.com' || uri.hostname == 'dl.suse.com')
```

Comment:
I'd suggest to do

```suggestion
      path.gsub!(%r{^/repo/\$RCE/}, '/\$RCE/') # drop '/repo' from SLE11 paths, to avoid double /repo/repo in local storage path.
```
to avoid further hostname hardcoding.

Reply:
path.gsub!(%r{^/repo/\$RCE/}, '/$RCE/')

added the above.